### ISSUE TITLE
Use with Azure AD?

### ISSUE THREAD

**Issue by olavt** [Azure-Samples/active-directory-ios-swift-native-v2#3]
I tried to use this sample against my Azure AD tenant. I registered a native app using the Azure AD Portal. But, when I enter my Azure AD username / password I get the following error message: "AADSTS70001: Application '<My AppId>' is not supported for this API version."

How can this be used with an Azure AD tenant?

**Comment by brandwe** [comment]
@olavt Did you use the portal at https://apps.dev.microsoft.com? This is the portal that creates an application which can be used with our new v2 endpoint and MSAL.

**Comment by olavt** [comment]
Ok, I used the regular Azure Portal to register my app in my Azure AD tenant. That was the issue. I now registered the app at https://apps.dev.microsoft.com and now everything worked as expected.

Maybe a little more information about that step could be added to clarify. I though that portal was for other scenarios and it's a little confusing to have multiple places to register apps. It's also important to login with your Azure AD tenants account when registering the app at https://apps.dev.microsoft.com.
